drivers/SmartThings/matter-lock/profiles/lock-user-pin-schedule-battery.yml

```diff
@@ -0,0 +1,31 @@
+name: lock-user-pin-schedule-battery
```

Comment:
for completion, could we also add this profile with -batteryLevel?

Reply:
Sure. I will add the lock-user-pin-schedule-batteryLevel profile.